pygotham/users/models.py

```diff
@@ -20,6 +20,8 @@
     db.Column('role_id', db.Integer, db.ForeignKey('roles.id')),
 )
 
+user_talks = db.Table('speakers', Speaker.metadata, autoload=True)
```

Comment:
Moar nits: Table names are pluralized, so their corresponding object names should match: `users_talks`. (I realize the table name here is different, but it's serving the same purpose here.)